src/view/com/modals/ChangeHandle.tsx

```diff
@@ -393,15 +393,15 @@ function CustomHandleForm({
       <View style={[styles.selectableBtns]}>
         <SelectableBtn
           selected={isDNSForm}
-          label="DNS Panel"
+          label={_(msg`DNS Panel`)}
```

Comment:
@pfrazee Should we localize this?

Reply:
Originally, the account handle was pinned after, but due to localization, I needed to include it in the `<Trans>` to allow us to reverse the order.